tests/bluetooth/bsim/audio/src/tbs_client_test.c

```diff
@@ -95,22 +103,128 @@ static void tbs_client_read_ccid_cb(struct bt_conn *conn, int err,
 	SET_FLAG(ccid_read_flag);
 }
 
+static void tbs_client_originate_call_cb(struct bt_conn *conn, int err,
+					 uint8_t inst_index,
+					 uint8_t call_index)
+{
+	printk("Client terminated call %u:\n", call_index);
+	call_placed = true;
+}
+
+static void tbs_client_hold_call_cb(struct bt_conn *conn, int err,
+				    uint8_t inst_index,
+				    uint8_t call_index)
+{
+	printk("%s\n", __func__);
+	if (err != 0) {
+		FAIL("Client hold call error: (%d)\n", err);
+		return;
+	}
+
+	printk("Client hold callback\n");
+	printk("Instance: %u\n", inst_index);
+	printk("Call index: %u\n", call_index);
+}
+
+static void tbs_client_retrieve_call_cb(struct bt_conn *conn, int err,
+				    uint8_t inst_index,
+				    uint8_t call_index)
+{
+	printk("%s\n", __func__);
+	if (err != 0) {
+		FAIL("Client retrieve call error: (%d)\n", err);
+		return;
+	}
+
+	printk("Client retrieve call callback\n");
+	printk("Instance: %u\n", inst_index);
+	printk("Call index: %u\n", call_index);
+}
+
+static void tbs_client_technology_cb(struct bt_conn *conn, int err,
+				    uint8_t inst_index,
+				    uint32_t value)
+{
+	printk("%s\n", __func__);
+	if (err != 0) {
```

Comment:
I'd suggest an empty line to separate the print and the error check. 

```suggestion
	printk("%s\n", __func__);
	
	if (err != 0) {
```